bucket/amfora.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.8.0",
+    "description": "Amfora aims to be the best looking Gemini client with the most features... all in the terminal",
+    "homepage": "https://github.com/makeworld-the-better-one/amfora",
+    "license": {
+        "identifier": "GPL-3.0-only",
+        "url": "https://github.com/makeworld-the-better-one/amfora/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/makeworld-the-better-one/amfora/releases/download/v1.8.0/amfora_1.8.0_windows_64-bit.exe#/amfora.exe",
+            "hash": "53c9dd483078455885016a2d4f94ab6f0d3ca145c631a086ae188791ca20c97d"
+        },
+        "32bit": {
+            "url": "https://github.com/makeworld-the-better-one/amfora/releases/download/v1.8.0/amfora_1.8.0_windows_32-bit.exe#/amfora.exe",
+            "hash": "bb8e3c197a8d8692a974997296859f5f069a128d05f6726463fdfd68a403bdd7"
+        }
+    },
+    "bin": "amfora.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/makeworld-the-better-one/amfora/releases/download/v$version/amfora_$version_windows_64-bit.exe"

```

Comment:
```suggestion
                "url": "https://github.com/makeworld-the-better-one/amfora/releases/download/v$version/amfora_$version_windows_64-bit.exe#/amfora.exe"
```

Reply:
not renaming it prevents amfora being added to PATH:
Creating shim for 'amfora'.
ERROR Cannot shim 'amfora.exe': File does not exist
Please contact the manifest maintainer!

Reply:
Can you read? This add the rename to the autoupdate...

After autoupdate, it will be replaced without renaming and fail installation..

Reply:
oh my bad. sorry i'm new to github